src/components/molecules/SfOptions/SfOptions.js

```diff
@@ -0,0 +1,51 @@
+import SfLabel from "../../atoms/SfLabel/SfLabel.vue";
+
+export default {
+  name: "SfOptions",
+  components: {
+    SfLabel
+  },
+  model: {
+    prop: "selected",
+    event: "change"
+  },
+  props: {
+    options: {
+      type: Array,
+      required: true
+    },
+    label: {
+      type: String,
+      default: null
+    },
+    labelClass: {
+      type: String,
+      default: null
+    },
+    image: {
+      type: Boolean,
+      default: false
+    },
+    color: {
+      type: Boolean,
+      default: false
+    },
+    selected: {
+      type: String,
+      default: null
+    },
+    optionClass: {
+      type: String,
+      default: null
+    },
+    wrapperClass: {
```

Comment:
i think this should be handled purely by css

Reply:
Thanks, I will fix it soon.